vendor/github.com/openshift/library-go/pkg/operator/loglevel/util.go

```diff
@@ -0,0 +1,103 @@
+package loglevel
+
+import (
+	"flag"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog/v2"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+)
+
+// LogLevelToVerbosity transforms operator log level to a klog numeric verbosity level.
+func LogLevelToVerbosity(logLevel operatorv1.LogLevel) int {
+	switch logLevel {
+	case operatorv1.Normal:
+		return 2
+	case operatorv1.Debug:
+		return 4
+	case operatorv1.Trace:
+		return 6
+	case operatorv1.TraceAll:
+		return 8
+	default:
+		return 2
```

Comment:
I agree that we probably shouldn't throw an error error. But, I'm thinking it might be worthwhile if we log it, just to help us understand a possible future state where a new log level is added, used, but doesn't manifest itself (as unlikely that is likely to be XD)

Reply:
Or maybe even throw an error and publish an event? Then the decision to use Normal for unknown is a bit higher up the stack?

Reply:
Sounds reasonable! Should we make a PR in library-go for this? We could also write our own helper, but then we'd be maintaining it.

Reply:
I can poke at that. Been cooking up one with k8s lib update for OpenShift/api to help clean up Lance's PR